skipruntime-ts/server/src/server.ts

```diff
@@ -5,11 +5,43 @@ import {
 } from "skip-wasm";
 import { createRESTServer } from "./rest.js";
 
+/**
+ * Initialize and start a service.
+ *
+ * Calling `runService` will initialize a service based on the `service` specification and start listening for requests on the specified `port`.
+ * This is a REST service that provides access to the internal reactive service.
+ * The service exposes some resources specified by a {@link SkipService}.
+ * Each resource has a _name_ and identifies a collection that associates _keys_ to _values_.
+ * The service responds to the following HTTP requests:
+ * - Read: `GET /v1/:resource/:key?param1=value1&...&paramN=valueN`
```

Comment:
Exactly how the query parameters are parsed is not completely clear to me, but specifying this much seems safe. AFAIU what really happens is that express just parses them in its default way, subject to `app.use(express.json());`, and that uses a parser "based on" the `qs` package, but without documenting what actually happens.

Reply:
This changes in the new API, where instantiating a resource is now made using a `POST /v1/streams` with a JSON body looking like:
```json
{
  "resource": "foo",
  "params": {
    ...
  }
}
```